### DISCUSSION TITLE
到底应该如何为指定输入法修改键盘布局？

### DISCUSSION THREAD

**Discussion by GrisMouche** [osfans/android-rime#1531] [Q&A | 问答] [answered]
我使用[五笔86](https://github.com/rime/rime-wubi)，按照教程帖上的说明在\rime\根目录下面创建trime.custom.yaml，在里面打上补丁

`patch:`
`  "preset_keyboards/wubi86/import_preset": qwerty0`

trime里重新部署。完成后看到trime.yaml里的确展开有

`wubi86:`
`  import_preset: qwerty0`

但实际在输入法中却什么反应也没有，仍然是默认的26键布局。
我也试过建立wubi86.schema.yaml的补丁wubi86.custom.yaml，输入同样的内容，也是无效。
也试过直接在build目录下的trime.yaml里把键盘布局命名里的qwerty跟qwerty0对调，结果也没有什么反应。
很苦恼。

**Comment by GrisMouche** [comment, marked as answer]
把已部署好的build删掉，用改好的trime.custom.yaml重新部署一遍就好了

**Reply by linzianabc** [reply]
"preset_keyboards/iridium_bpmf/import_preset": bopomofo   #预设36键布局的ID是qwerty0

**Comment by Endif12** [comment]
意思是说现在要加载trime.custom.yaml必须先清空build吗？我也遇到最近的版本不加载我的trime.custom.yaml的问题，回去试试看
